content/use-cases/ebr.md

```diff
@@ -0,0 +1,175 @@
+---
+title: >-
+  Electronic batch records
+description: An example of how to use Rhize to create Electronic Batch Records for pharmaceutical manufacturing
+categories: ["howto", "use-cases"]
+weight: 001
+menu:
+  main:
+    parent: use-cases
+    identifier:
+---
+
+In pharmaceutical manufacturing, an _Electronic Batch Record (eBR)_ is a detailed report about a specific batch.
+It includes information about items such as the process segments, material, personnel, and equipment.
+
+As Rhize can ingest data from all levels of the manufacturing process, and do so in real-time, it is an ideal single source of truth to create eBRs.
+The process can be automated and highly efficient.
+Our experience in production systems shows that Rhize can typically generate an eBR in less than 500 milliseconds. 
+
+The procedure has the following steps:
+
+1. Identify the sources of data.
+1. Map the fields for these data sources to Rhize's ISA-95 schema.
+1. Write a {{< abbr "BPMN" >}} process that listens for a data source, transforms the incoming data to the schema, and then sends a {{< abbr "mutation" >}} to update the Rhize database.
```

Comment:
It is very rare that this will be only 1 process. Suggest we change to:

`1. Write {{< abbr "BPMN" >}} processes that listen to data sources, transforms the incoming data to the schema, and then sends a {{< abbr "mutatino" >}} to update the Rhize database.